README.md

```diff
@@ -86,6 +86,19 @@ We need to know some info about hosts, but not much.  See the files for details,
 
 This is a description of all the roles that are included by installing subspace, along with their configuration.
 
+## Procedure for updating on projects
```

Comment:
This whole section should go higher up. I think at the bottom of the Usage section.